src/components/GenreSelect/GenreList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './GenreList.scss';
+
+class GenreList extends React.Component {
+    handleSelect = (index) => {
+        this.props.onSelect(index);
+    }
+    render() {
+        return (
+            <ul className="GenreSelect__list">
+                {this.props.genreList.map((item, index) => {
+                    return <li
+                        key={index}
```

Comment:
It is not a good practise to use index as a key. Please create unique id in genreList for each genre and use it as a key.

Useful links:
https://react.dev/learn/rendering-lists
https://www.youtube.com/watch?v=xlPxnc5uUPQ

Reply:
![image](https://github.com/mradomski-epam/reactjs-mentoring/assets/118353174/097f68b0-cc4b-42f2-901c-50699f592a6e)